src/reference/informative/informA1.rst

```diff
@@ -8,4 +8,32 @@
 
   .. container:: infospec
 
-    **TODO** Informal notes go here
+    .. container:: heading3
+
+      Namespaces
+
+    Namespaces are a way of making sure that names and definitions are interpreted within the right frame of reference.  In CellML, two namespaces are used.  These are the CellML namespace itself (which helps to define the :code:`units` elements as distinct from the XML default units system) and the MathML namespace used to interpret the :code:`<math>` elements.  For more information on namespaces in general, please refer to the :namespace_help:`W3 schools namespace page <>`.  For more information on the MathML namespace please refer to the :mathml2help:`W3 MathML namespace <#inferf.namespace>` page.
+    
+    .. container:: heading3
+
+      Unicode, Basic Latin, and European numerals
+
+    The Unicode project is an attempt to codify all of the symbols - alphabets, writings, even emojis - of all the languages of the world so that they can be interchangeable and interpreted by computers.  Since computers understand numbers rather than symbols, each character in each language is given an unique numerical code.  The codes themselves are arranged into blocks representing sets or alphabets of characters, and the Basic Latin alphabet is one of these.  It contains the upper and lowercase alphabet without decoration:
+
+      :code:`abcdefghijklmnopqrstuvwxyz` (symbols between U+0061 and U+007A)
+      :code:`ABCDEFGHIJKLMNOPQRSTUVWXYZ` (symbols between U+0041 and U+005A)
```

Comment:
Might want to have a link to U+0061, etc. (same below)

Reply:
There are those links in the main text already.  Not really required to have them in every single place ...?

Reply:
I like to think of the main text and the "See more" section as being two different things. Some people might just go straight to the "See more" section, so I would expect links, the first time round at least.

Personally, should `U+0061` be mentioned many times then I would have it as a link everywhere. I just hate having to scroll up/down just so that I can find where the link was used.

Reply:
> I like to think of the main text and the "See more" section as being two different things.

Sorry @agarny, but this is not how it's been written at all!  It was written intended to be embedded within the context of the main text, so it refers to the main text frequently, and certainly doesn't make any sense as a standalone document.  It's intended to expand upon and explain the main document, not replace it entirely.  
I'll add the links here, but in general (as in our previous discussions of the current libcellml docs) I find having links all over the place can get really cluttered and confused.  I've tended to use the link in the same way you might define an acronym or special term - define it/link to it on first use, then expect that the reader will have accessed that the first time as they read it, should they need to.

Reply:
48a9597

Reply:
> Sorry @agarny, but this is not how it's been written at all! It was written intended to be embedded within the context of the main text, so it refers to the main text frequently, and certainly doesn't make any sense as a standalone document. It's intended to expand upon and explain the main document, not replace it entirely.

Indeed, fair enough.

> I'll add the links here, but in general (as in our previous discussions of the current libcellml docs) I find having links all over the place can get really cluttered and confused. I've tended to use the link in the same way you might define an acronym or special term - define it/link to it on first use, then expect that the reader will have accessed that the first time as they read it, should they need to.

I like to be able to click on things wherever they are, but maybe that's just me although I do appreciate the potential clutter issue.